source/Core/Module/ModuleTemplateBlockRepository.php

```diff
@@ -92,12 +90,8 @@ public function getBlocks($shopTemplateName, $activeModulesId, $shopId, $themesI
      */
     private function formActiveThemesIdQuery($activeThemeIds = [])
     {
-        $defaultThemeIndicator = '';
```

Comment:
Not sure here. Before variable said what it is, and now I see only empty string. How do you think?